modules/gui/src/com/haulmont/cuba/gui/app/core/categories/CategoryBrowser.java

```diff
@@ -134,4 +145,27 @@ public void actionPerform(Component component) {
             }
         }
     }
+
+    protected void initAttrDataTypeColumn() {
+        attributesTable.removeGeneratedColumn("dataType");
+        attributesTable.addGeneratedColumn("dataType", attribute -> {
+            Label dataTypeLabel = componentsFactory.createComponent(Label.class);
+            String labelContent;
+            if (BooleanUtils.isTrue(attribute.getIsEntity())) {
+                Class clazz = attribute.getJavaClassForEntity();
+
+                if (clazz != null) {
+                    MetaClass metaClass = metadata.getSession().getClass(clazz);
+                    labelContent = messageTools.getEntityCaption(metaClass);
+                } else {
+                    labelContent = "classNotFound";
```

Comment:
if class is null, show empty string. Because `classNotFound` isn't localized value.